platform-ui-skin/src/main/webapp/skin/less/helpers.less

```diff
@@ -315,6 +315,14 @@
     max-width: 100%;
     margin: auto;
 }
+.text-truncate-1 {
+  display: -webkit-box;
+  -webkit-line-clamp: 1;
+  -webkit-box-orient: vertical;
+  text-overflow: ellipsis ;
+  overflow: hidden ;
+  white-space: initial ;
+}
```

Comment:
```suggestion
```
Not needed class, please use simply `text-truncate` when it's for one simple line.

Reply:
with text-truncate, we are sometimes forced to add a maximum width for it to work.
In this case, we are not

Reply:
ok, so let's add `text-truncate width-max-content` on HTML elements instead of defining a new class with same behavior as `text-truncate`.

Reply:
max-width should be in pixels and it depends on the element 
in this case, with "-webkit-line-clamp: 1"  it will be re-usable

Reply:
nope, not true, why `max-width` should be in pixels only ?

Reply:
tested since we have already a class for width-max-content in helpers